ext/standard/basic_functions.stub.php

```diff
@@ -408,23 +408,23 @@ function dns_check_record(string $hostname, string $type = "MX"): bool {}
 function checkdnsrr(string $hostname, string $type = "MX"): bool {}
 
 /**
- * @param array $authns
- * @param array $addtl
+ * @param array $authoritative_name_servers
+ * @param array $additional_records
  */
-function dns_get_record(string $hostname, int $type = DNS_ANY, &$authns = null, &$addtl = null, bool $raw = false): array|false {}
+function dns_get_record(string $hostname, int $type = DNS_ANY, &$authoritative_name_servers = null, &$additional_records = null, bool $raw = false): array|false {}
```

Comment:
These names are excessively long, but I'm not familiar enough with this API to suggest a good concise name.